pkg/clusters/addons/kong/kong.go

```diff
@@ -29,6 +30,21 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongLicenseSecretName is the kong license data secret name
+	KongLicenseSecretName = "kong-enterprise-license"
+
+	// EnterpriseAdminPasswordSecretName is the kong admin seed password
+	EnterpriseAdminPasswordSecretName = "kong-enterprise-superuser-password"
+
+	// EnterprisePWD is password
+	EnterprisePWD = "password"
```

Comment:
If we're going to provide a password (rather than providing a method like `WithSuperuserPassword(password string)` to allow the caller to pass it in), we should generate that password instead of using the default password "password". Even though this is for testing environments, it's trivial to generate a password. Here's an example of generating passwords in tests from KIC:

https://github.com/Kong/kubernetes-ingress-controller/blob/5d724746ca3c7fec6472f71681a4ec76932356fc/test/e2e/all_in_one_test.go#L394

In fact in this case coincidentally the exact purpose was to generate a superuser password for Kong.

Reply:
Its a great idea generating password 👍 . 
This Kong Admin password is needed for KIC to add super-admin, regular RBAC user, and only for test purpose. We dont want to copy the generated password from the log and use in the KIC.

Reply:
No we don't want to copy the generated password that's not what I was suggesting. I was suggesting we generate a unique password every time, and don't provide _any_ default.

Reply:
https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-7cc1dbd395c3c0ea6e62a7d5a3d4493f37054d7b7be23f198340cdd6a52c0be2R68

Reply:
Add Interface https://github.com/Kong/kubernetes-testing-framework/pull/104
and test case
7939042001b26a851c084992080dd4b78b4344cf